DevAcademy/Scenes/Favorites/FavoritesPlacesView.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+struct FavoritePlacesView: View {
+    @Environment(\.colorScheme) var colorScheme
+    
+    var body: some View {
+        NavigationStack {
+            ZStack {
+                Color.theme.accent
+                VStack(spacing: 100) {
+                    Text("Tvoje oblíbená místa")
+                        .font(.title)
+                        .fontWeight(.medium)
+                        .lineLimit(1)
+                     
+                    Image(systemName: "star")
+                        .resizable()
+                        .renderingMode(.template)
+                        .frame(width: 150, height: 150)
+                        .fontWeight(.ultraLight)
+                }
+                .foregroundColor(
+                    Color(colorScheme == .dark ? .black : .white))
```

Comment:
Tohle je jedna cesta jak pracovat s tim dark modem. Ale je tu mozna jeste jednodussi cesta a to kdyz si do palety (`Assets`) pridas vlastni `black` barvu, ktere nastavis rozdilne Appearance a v Dark appearance ji nastavis bilou.
Pak budes moct pouzit pouze `Color.theme.black` a nebudes si muset na vsech mistech vytahovat to `.colorScheme`.

A jeste vlastne pro tip, kdyz uz by to vlastne byla cerna, ktera je v dark modu bila, lepsi pojmenovani by bylo trosku univerzalnejsi pojmenovani, treba `ink`, aby to nebylo zavadejici, ze v kodu budes videt `.black` ale bude to bile. 😉 

![Screenshot 2023-08-14 at 10 17 08](https://github.com/PavlaBerankova/dev-academy-ios/assets/12135948/74238fe6-4474-4706-b631-94e12c6e8bad)